### PR TITLE
Alex supriana

### PR DESCRIPTION
**Pull request by asupriana**

### CODE REVIEW

**Comment by asupriana** [comment]
Finally finished stretch!

File: Answers.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+
 1.  Which HTTP methods can be mapped to the CRUD acronym that we use when interfacing with APIs/Servers.
-1.  Mention three tools we can use to make AJAX requests
\ No newline at end of file
+Create - POST
+Read - GET
+Update - PUT
+Delete - DELETE
+
+1.  Mention three tools we can use to make AJAX requests
```
Comment (PhilHart23) [inline Answers.md]: 5/5

File: village/src/App.js [hunk 2]
```diff
@@ -14,11 +15,52 @@ class App extends Component {
...
+      .delete(`http://localhost:3333/smurfs/${id}`)
+      .then(res => {
+        this.setState({smurfs: res.data});
+      })
+      .catch(error => {
+        console.error('NO!', error);
+      });
+  };
+
+
```
Comment (PhilHart23) [inline village/src/App.js]: Perfect!

File: village/src/App.js [hunk 3]
```diff
@@ -14,11 +15,52 @@ class App extends Component {
...
+          </NavLink>
+        </div>
+        <Route
+        exact path="/smurf-form"
+        component={SmurfForm}
+        />
+        <Route path="/"
+         render={props => (
+          <Smurfs
+          {...props}
```
Comment (PhilHart23) [inline village/src/App.js]: Great use of the spread operator

File: village/src/components/SmurfForm.js [hunk 4]
```diff
@@ -54,4 +56,16 @@ class SmurfForm extends Component {
...
+    age: PropTypes.number,
+    height: PropTypes.string
+  }),
+  addSmurf: PropTypes.func,
+  handleInputChange: PropTypes.func
+}
+
+
+
 export default SmurfForm;
```
Comment (PhilHart23) [inline village/src/components/SmurfForm.js]: Great job!

**Comment by PhilHart23** [comment]
Great work on this sprint!

**Pull request merged**